app/index.ios.js

```diff
@@ -5,6 +5,13 @@ var MOCKED_CHAT_DATA = [
   {member: 'Richard', message: 'To the apple store!'}
 ];
 
+var pubnub = require("pubnub")({
+    ssl           : true,  // <- enable TLS Tunneling over TCP 
+    channel : 'CrewAppChannel',
+    publish_key   : "pub-c-ed3f13a5-4e20-4ea5-8760-1fce2f13b370",
```

Comment:
This is extremely bad practice... please don't add api key's to a repo on GitHub